src/server/console/logind_console_services.cpp

```diff
@@ -159,7 +162,7 @@ std::unique_ptr<LogindSeat, decltype(&g_object_unref)> simple_seat_proxy_on_syst
         }).get();
 }
 
-std::string object_path_for_current_session(LogindSeat* seat_proxy)
+std::string object_path_for_current_session(LogindSeat *seat_proxy)
```

Comment:
There are a lot of spurious, whitespace changes to this file. A lot of them away from project style